src/sidebar/components/login-control.js

```diff
@@ -1,16 +1,48 @@
 'use strict';
 
+var bridgeEvents = require('../../shared/bridge-events');
 var persona = require('../filter/persona');
+var serviceConfig = require('../service-config');
 
 module.exports = {
   controllerAs: 'vm',
+
   //@ngInject
-  controller: function (serviceUrl, settings) {
+  controller: function (bridge, serviceUrl, settings, $window) {
```

Comment:
Nit: Built-in services should appear at the start of the parameter list (ie. `$window, bridge, ...`).